### ISSUE TITLE
Legacy Issue: All edited in mentions are counted as user mentions

### ISSUE THREAD

**Issue by JamesLongman** [dignityofwar/digbot#29]
Original issue by James:
> If someone tries to edit in a mention either legitimately or attempting to circumvent the limit we still detect and count it. However, we're unable to distinguish between user and role mentions atm.

This was a discord.js bug and has been fixed in more recent versions. Bumping discord.js may fix this without any need for other edits in code

**Comment by microwavekonijn** [comment]
I was looking at this and I noticed a few things. First on the message object has a mentions object with collections of all the mentions where each collection has a size property which corresponds with the unique amount of mentions(note: on that object users and members are the same, sort of).

I also found that if you mention someone it will trigger a notification, but if you edit the message and then add a mention it will not trigger a notification. The opposite is also true, where someone mentions someone then edits it and removes the mention the notification will still remain. So I don't think it is necessary to check the edited messages in the mentionspam module.
